.circleci/config.yml

```diff
@@ -167,47 +166,52 @@ commands:
           key: << parameters.key_prefix >>-{{ .Branch }}
 
   manage_api_cache:
-    description: "workbench/api: save, restore or save_and_restore gradle cache"
+    description: "workbench/api: save or restore gradle cache"
     parameters:
       save:
         type: boolean
         default: false
       restore:
         type: boolean
         default: false
-      key_prefix:
-        type: string
-        default: v1-api-cache-{{ checksum "~/workbench/api/build.gradle" }}
     steps:
       - when:
           condition: << parameters.save >>
           steps:
+            # splitting into multiple distinct caches. Refers to https://circleci.com/docs/2.0/caching/#cache-size
             - save_cache:
                 paths:
-                  - ~/.gradle/caches
                   - ~/.gradle/wrapper
-                  - ~/.m2
-                  - ~/workbench/api/build/exploded-api/WEB-INF/lib/
```

Comment:
I am not sure either dir existed to have anything to cache. Am I wrong?

Reply:
The deployment job puts stuff in exploded-api
~/.m2 may contain maven caches, but unsure

You should be able SSH into a circle job and check to confirm